src/packagedcode/utils.py

```diff
@@ -146,3 +147,22 @@ def build_description(summary, description):
             description = '\n'.join([summary , description])
 
     return description
+
+def combine_expressions(expressions, licensing=Licensing()):
+    """
+    Return a combined license expression string with AND, given a list of
+    license expressions.
+
+    For example:
+    >>> a = 'mit'
+    >>> b = 'gpl'
+    >>> combine_expressions([a, b])
+    'mit AND gpl'
+    """
+    if not expressions:
+        return
+    if len(expressions) == 1:
```

Comment:
what happens if expressions is `q` ? this tests alone does not make sense to me.

Reply:
@pombredanne Can you please explain more about q? Do you mean non list case? :)

Reply:
what if this expression is only a single character string and not a list?